docs/Learning-Environment-Design-Academy.md

```diff
@@ -31,7 +31,7 @@ in your Academy subclass.
 Implement an `AcademyReset()` function to alter the environment at the start of
 each episode. For example, you might want to reset an Agent to its starting
 position or move a goal to a random position. An environment resets when the
-Academy `Max Steps` count is reached.
+Python training process needs to reset the environment to facilitate training.
```

Comment:
Maybe be more explicit here and say when `env.reset()` is called, since technically a reset could be called outside of the context of training.

Reply:
Edited it.